.github/workflows/CI.yml

```diff
@@ -76,7 +76,6 @@ jobs:
       - name: Install environment with ${{ env.JOBNAME }}
         uses: mamba-org/setup-micromamba@v1
         with:
-          micromamba-version: '1.5.10-0'
```

Comment:
These are duplicate lines from a bad merge somewhere and were causing syntax errors